.jenkins/multigpu-test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+COMPACT_JOB_NAME="${BUILD_ENVIRONMENT}-multigpu-test"
+source "$(dirname "${BASH_SOURCE[0]}")/common.sh"
+
+# Required environment variable: $BUILD_ENVIRONMENT
+# (This is set by default in the Docker images we build, so you don't
+# need to set it yourself.
+
+export PATH=/opt/conda/bin:$PATH
+
+export LD_LIBRARY_PATH=/usr/local/cuda/lib64/stubs:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=/usr/local/cuda/lib64:$LD_LIBRARY_PATH
+
+echo "Testing pytorch"
+export OMP_NUM_THREADS=4
+export MKL_NUM_THREADS=4
+
+time python test/run_test.py --verbose -i distributed
```

Comment:
What makes it a "multigpu-test"? It looks like it's just running a subset of our tests without doing anything special. Is it treated differently by our CI infra?

Reply:
I think it's not treated differently. Just that for most test cases, it doesn't worth testing again on 4GPU machines since they exercise exactly the same code path as 2GPU machines. So here we only run the distributed test on 4GPU machines. Correct me if I'm wrong @ezyang @apaszke .

Reply:
That sounds as if it was treated specially by our CI system by running it on a different set of machines than the one used for regular tests (where we don't use 4 GPUs). Is that right?

Reply:
Yes.